docs/AppService-dotnetcore.md

```diff
@@ -0,0 +1,58 @@
+# Deploy Helium to Azure App Service
+
+Create and configure App Service on Windows using dotnetcore 3.1
+
+```bash
+
+# create App Service plan
+az appservice plan create --sku S1 -g $He_App_RG -n ${He_Name}-plan
+
+# create Web App
+az webapp create --runtime "dotnetcore|3.1" -g $He_App_RG -p ${He_Name}-plan -n $He_Name
+
+# stop the Web App
+az webapp stop -g $He_App_RG -n $He_Name
+
+# assign Managed Identity
+export He_MI_ID=$(az webapp identity assign -g $He_App_RG -n $He_Name --query principalId -o tsv)
+
+# grant Key Vault access to Managed Identity
+az keyvault set-policy -n $He_Name --secret-permissions get list --key-permissions get list --object-id $He_MI_ID
+
+# set the Key Vault config setting
+az webapp config appsettings set --settings KEYVAULT_NAME=$He_Name -g $He_App_RG -n $He_Name
+
+# delete the deployment source if it exists
+az webapp deployment source delete -g $He_App_RG -n $He_Name
+
+# set deployment
+az webapp deployment source config -g $He_App_RG -n $He_Name -u https://github.com/retaildevcrews/helium-csharp --branch main
```

Comment:
Was able to run the rest of the steps successfully besides this one. I'm getting an error `Cannot find SourceControlToken with name GitHub` while running from codespaces. Not sure if you've ran into this error, could be an issue with my setup.

Reply:
@retaildevcrews/helium-team - I didn't run into this issue but wasn't using codespaces - can someone else test on codespaces to see if it's a codespace issue please?

Reply:
Tried it on my local machine, same error. This is odd.

Reply:
let me test again - could be a typo.

Are you an owner on the subscription? It creates an SSH token (I get the emails), so if you're not an owner, it may not work.

Reply:
Yup, I've got the Owner role in this subscription. I've made some progress by granting access for App Service to my Github Account in the `Deployment Center` sidebar item. Now running into a creating webhook issue and which can be explained by [this FAQ item](https://github.com/projectkudu/kudu/wiki/Investigating-continuous-deployment#q-cant-see-the-repositories-in-the-drop-down).

Reply:
Could be org permission issues ... can you fork to a personal repo and try that?

Reply:
Ah that did the trick, awesome!

Reply:
Excellent - I just pushed an update to the doc calling this out. We'll do some more investigating and testing and may need to file CSE feedback. I'll update the issue on the board.